src/main/java/picard/util/LiftOverIntervalList.java

```diff
@@ -137,22 +135,27 @@ protected int doWork() {
         IOUtil.assertFileIsReadable(SEQUENCE_DICTIONARY);
         IOUtil.assertFileIsReadable(CHAIN);
         IOUtil.assertFileIsWritable(OUTPUT);
+        if (REJECT != null) IOUtil.assertFileIsWritable(REJECT);
```

Comment:
I see you're just following the existing pattern, but... Would all these asserts be more appropriate in an `@Override` of `customCommandLineValidation`?

Reply:
this is not only the patterns in this tool...but all over picard. I'm not saying what's appropriate, but I think it's a larger discussion....please file an issue if you feel strongly about this.